src/main/java/spring/service/UserServiceImpl.java

```diff
@@ -24,4 +25,9 @@ public void add(User user) {
     public List<User> listUsers() {
         return userDao.getAll();
     }
+
+    @Override
+    public Optional<User> get(Long id) {
```

Comment:
```suggestion
    public User get(Long id) {
```
service should return an entity or throw an exception(you can user orElseThrow for this)